mlflow/store/sqlalchemy_store.py

```diff
@@ -0,0 +1,200 @@
+import sqlalchemy
+from sqlalchemy import orm
+from sqlalchemy.exc import IntegrityError
+from mlflow.store.dbmodels import models
+from mlflow import entities
+from mlflow.entities import ViewType
+from mlflow.exceptions import MlflowException
+import mlflow.protos.databricks_pb2 as error_codes
+
+
+class SqlAlchemyStore(object):
+
+    def __init__(self, db_uri='sqlite://'):
```

Comment:
Can we change this init function to require a connection string rather than defaulting to `sqllite`?